### PR TITLE
Adding information in the commandline tool help to prevent accidental `discharge` during clamshell mode

### PR DESCRIPTION
**Pull request by iamchanghyunpark**
`battery discharge` should not be used with clamshell mode. (Laptop just goes to sleep) These two commits add information to the `battery` commandline tool output and the `README.md` that also prints the output of the `battery` commandline tool that warns against using `battery discharge` while in clamshell mode.

### CODE REVIEW

**Comment by actuallymentor** [comment]
Ah nice one. Could you rephrase it to "does not work properly in clamshell mode" for clarity?

**Comment by iamchanghyunpark** [comment]
Sorry, the update should have been a single commit!
Is this better?
